cmd/machine.go

```diff
@@ -5,10 +5,10 @@ import (
 	"fmt"
 	"os"
 
+	packet_logr "github.com/packethost/pkg/log/logr"
```

Comment:
oh this is weird to me, why this instead of just `logr`?

Reply:
ah, I can change it. I think I had `https://github.com/go-logr/logr` in here at some point and needed to de-conflict.